includes/multi-currency/class-frontend-currencies.php

```diff
@@ -31,9 +38,11 @@ class Frontend_Currencies {
 	 * Constructor.
 	 *
 	 * @param Multi_Currency $multi_currency The Multi_Currency instance.
+	 * @param Utils          $utils          The Utils instance.
 	 */
-	public function __construct( Multi_Currency $multi_currency ) {
+	public function __construct( Multi_Currency $multi_currency, Utils $utils ) {
```

Comment:
This is a nitpick, but what do you think about adding return type hints (where we are able to) to the functions in this class? :)

Reply:
Sure! I've added them in https://github.com/Automattic/woocommerce-payments/pull/2312/commits/4216424bdabeeb6a3d6f8cce71ec0f5462b1ca9e, and also fixed some wrong types in the comments.

Reply:
Awesome, thanks for making that change!